flash/core/data/utils.py

```diff
@@ -34,16 +34,15 @@ def download_file(url: str, path: str, verbose: bool = False) -> None:
     if not os.path.exists(path):
         os.makedirs(path)
     local_filename = os.path.join(path, url.split('/')[-1])
-    r = requests.get(url, stream=True)
-    file_size = int(r.headers['Content-Length']) if 'Content-Length' in r.headers else 0
-    chunk = 1
-    chunk_size = 1024
-    num_bars = int(file_size / chunk_size)
-    if verbose:
-        print(dict(file_size=file_size))
-        print(dict(num_bars=num_bars))
 
     if not os.path.exists(local_filename):
+        r = requests.get(url, stream=True)
+        file_size = int(r.headers['Content-Length']) if 'Content-Length' in r.headers else 0
```

Comment:
```suggestion
        file_size = int(r.headers.get('Content-Length', 0))
```

Reply:
This should be a configurable option I can think of use cases where I might want to redownload if the original file changes.